numba/tests/test_stencils.py

```diff
@@ -365,12 +407,1272 @@ def test_impl(n):
+        def kernel(a, b):
+            s = 1
+            acc = 0 
+            for k in b[0, :]:
+                acc += k
+            t = 2 - s - 1
+            return a[0, 1] + b[0, t] + acc
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        self.check(kernel, a, b, options={'standard_indexing': 'b'})
+        
+    def test_basic57(self):
+        """2 args, standard_indexing, split index operation """
+        def kernel(a, b):
+            c = b[0]
+            return a[0, 1] + c[1]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        self.check(kernel, a, b, options={'standard_indexing': 'b'})
+
+    # parfors fails
+    def test_basic58(self):
+        """2 args, standard_indexing, split index with broadcast mutation """
+        def kernel(a, b):
+            c = b[0] + 1
+            return a[0, 1] + c[1]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        self.check(kernel, a, b, options={'standard_indexing': 'b'})
+
+    def test_basic59(self):
+        """3 args, mix of array, relative and standard indexing and const"""
+        def kernel(a, b, c):
+            return a[0, 1] + b[1, 1] + c
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        c = 10
+        self.check(kernel, a, b, c, options={'standard_indexing': ['b', 'c']})
+
+    # parfors fails
+    def test_basic60(self):
+        """3 args, mix of array, relative and standard indexing, tuple pass through"""
+        def kernel(a, b, c):
+            return a[0, 1] + b[1, 1] + c[0]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        c = (10,)
+        self.check(kernel, a, b, c, options={'standard_indexing': ['b', 'c']})
+
+    def test_basic61(self):
+        """2 args, standard_indexing on first"""
+        def kernel(a, b):
+            return a[0, 1] + b[1, 1]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        self.check(kernel, a, b, options={'standard_indexing': 'a'})
+
+    def test_basic62(self):
+        """2 args, standard_indexing and cval"""
+        def kernel(a, b):
+            return a[0, 1] + b[1, 1]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12.).reshape(3, 4)
+        self.check(kernel, a, b, options={'standard_indexing': 'b', 'cval': 10.})
+
+    def test_basic63(self):
+        """2 args, standard_indexing applied to relative, should fail, non-const idx"""
+        def kernel(a, b):
+            return a[0, b[0, 1]]
+        a = np.arange(12.).reshape(3, 4)
+        b = np.arange(12).reshape(3, 4)
+        ex = self.exception_dict(pyStencil=ValueError, stencil=ValueError, parfor=ValueError, njit=LoweringError)
+        self.check(kernel, a, b, options={'standard_indexing': 'b'}, expected_exception=ex)
+        
+    # stencil, njit, parfors all fail. Does this make sense?
```

Comment:
Think this is/may be the same as https://github.com/IntelLabs/numba/pull/37/files#r145183180

Reply:
Looks like parfor raises the correct exception to me.